metadata.json

```diff
@@ -25,7 +25,7 @@
       "version_requirement": ">= 2.4.0 < 4.0.0"
     },
     {
-      "name": "herculesteam-augeasproviders_sysctl",
+      "name": "puppet-augeasproviders_sysctl",
```

Comment:
please also update https://github.com/voxpupuli/puppet-k8s/blob/master/.fixtures.yml to pull the correct repo (from git, not the forge)

Reply:
good catch!
I see camptocamp/systemd there.. shall I update it too?

Reply:
Updated

Reply:
yes please